graphql/graphql.go

```diff
@@ -415,13 +438,13 @@ func (b *Block) resolveReceipts(ctx context.Context) ([]*types.Receipt, error) {
 	return b.receipts, nil
 }
 
-func (b *Block) Number(ctx context.Context) (hexutil.Uint64, error) {
+func (b *Block) Number(ctx context.Context) (Long, error) {
 	header, err := b.resolveHeader(ctx)
 	if err != nil {
-		return 0, err
+		return Long(0), err
```

Comment:
explicit cast not needed, you can leave this as it was
```suggestion
		return 0, err
```